### ISSUE TITLE
안녕하세요 ROW 이동 이벤트 

### ISSUE THREAD

**Issue by shinheechul2** [ax5ui/ax5ui-grid#99]
안녕하세요 
ROW 를 수정할때 TEXT 나 SELECT 를 바꿀때마다 실시간으로 
저장을 하고싶은데

TEXT에  내용을 입력하고 
엔터나 마우스를 다른쪽으로 넘기면  SPAN 으로 바뀌는데 
이  이벤트를 체크 할수있을까요?

알려주시면 감사합니다 ^^

**Comment by thomasJang** [comment]
```js
{
  body:{
    onDataChanged: function(){
    }
  }
}
```

하시면 변경될 때 이벤트를 받을 수 있습니다.
그런데 변경될 셀만 저장하고 내부에 edited값을 초기화 하는 것은 고려가 안되서어요 필요하시다면 메소드를 추가해야 가능하겠습니다.

**Comment by shinheechul2** [comment]
네 감사합니다 ^^ 혹시 엔터 이벤트인지 마우스 포커스가 다른곳으로 이동했을떄 이벤트인지 구분할수있는 방법이 있을까요?
